### PR TITLE
Removed aria-disabled in NxCollapsibleItems - RSC-984

### PR DESCRIPTION
**Pull request by cindyko226**
https://issues.sonatype.org/browse/RSC-984

### CODE REVIEW

**Comment by cindyko226** [comment]
CI - Receiving a failed test about the position of disabled collapsible items.
nx-collapsible-items-js-nx-collapsible-items-disabled-nx-collapsible-items-looks-right-1-diff

**Comment by cindyko226** [comment]
According to the updates in Kaleido https://www.sketch.com/s/2393264e-1233-4909-99c8-a7dda02588cd/a/xr1Ojx0
The color of arrow and listed item is updated here 4a280e453368b83d8a680f6a57b3d641fa9360ee

**Comment by rpokorny** [comment]
> CI - Receiving a failed test about the position of disabled collapsible items.

I'll update that for you

File: lib/src/components/NxCollapsibleItems/NxCollapsibleItems.scss [hunk 1]
```diff
@@ -43,7 +43,7 @@ $-form-child-left-padding: calc(#{$-trigger-side-padding} + #{$-triangle-box-wid
 }
 
 .nx-collapsible-items--disabled {
-  .nx-collapsible-items__text, .nx-collapsible-items__child {
+  .nx-collapsible-items__text, .nx-collapsible-items__twisty {
```
Comment (rpokorny) [inline lib/src/components/NxCollapsibleItems/NxCollapsibleItems.scss:46]: The ".nx-collapsible-items__trigger` referenced in the selector below contains both the child and the twisty, so it'd be better to just move the `color` down there

**Comment by rpokorny** [comment]
> > CI - Receiving a failed test about the position of disabled collapsible items.
>
> I'll update that for you

bb9f7a8f1

File: lib/src/components/NxCollapsibleItems/NxCollapsibleItems.scss (hunk 1 above)
Comment (cindyko226) [reply lib/src/components/NxCollapsibleItems/NxCollapsibleItems.scss]: This is fixed here e031b37beb3d0c2e0a02591f351e5987b0f42c22, thanks

**Review by rpokorny — APPROVED** [review]

**Pull request merged**